lib/scr/flutterflow_cli_base.dart

```diff
@@ -0,0 +1,190 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:archive/archive_io.dart';
+import 'package:http/http.dart' as http;
+import 'package:path/path.dart' as path_util;
+
+const kDefaultEndpoint = 'https://api.flutterflow.io/v1';
+
+class FlutterFlowCli {
+  Future<String?> export({
+    required String token,
+    required String projectId,
+    required String destinationPath,
+    required bool includeAssets,
+    String endpoint = kDefaultEndpoint,
+    String? branchName,
+  }) =>
+      exportCode(
+        token: token,
+        endpoint: endpoint,
+        projectId: projectId,
+        destinationPath: destinationPath,
+        includeAssets: includeAssets,
+        branchName: branchName,
+        fix: false,
+      );
+}
```

Comment:
Why do we need this class? Can we just expose `exportCode()`?

Reply:
I guess we can do that, but I feel like it's better to have as a separate class to prevent `exportCode()` method from conflicting with any existing method with the same name inside a project.

Reply:
It's a matter of taste I guess, so up to you.

If you want to keep the class, then let's:
1) Add option for `fix`.
2) Rename it to `FlutterFlowApi`? Because it's not really "CLI" if you use it this way.